download.go

```diff
@@ -131,8 +129,11 @@ type DownloadConfig struct {
 
 type DownloadOption func(c *DownloadConfig)
 
-func WithUserAgent(ua string) DownloadOption {
+func WithUserAgent(ua, fallback string) DownloadOption {
 	return func(c *DownloadConfig) {
+		if ua == "" {
+			ua = fallback
```

Comment:
I don't think necessary the fallback variable.

Can't do like this?

```suggestion
			ua = "Pget/" + version
```